tabletop/core/manual_turn.js

```diff
@@ -58,13 +58,25 @@ function ManualTurn(game, startView, view, gameOverView, nextPlayerView) {
       waitingForMove: { 
         _onEnter: function() { 
           view.drawView();
-          console.log(this.game.getCurrentPlayer().name + ": Make your move.");
+          if (this.game.getCurrentPlayer().name == "AI") { 
+            var AIMove = this.game.pickAImove(this.game.getCurrentPlayer(), this.game);
+            game.proposedMove = AIMove;
+            this.handle("makeMove");
+          } else { 
+            console.log(this.game.getCurrentPlayer().name + ": Make your move.");
+          }
         },
         
         makeMove : function() { 
           if (game.hasValidMove()) { 
-            game.executeMove();
-            this.transition("postTurn");
+            if (this.game.getCurrentPlayer().name == "AI") { 
+              var turnMap = this;
+              setTimeout(function() { game.executeMove(); turnMap.transition("postTurn"); }, 500);
```

Comment:
for right now yeah. ideally we time analysis and if it's under .5 seconds then we add a delay. without a delay the move is made so quick it's hard to understand what happened